### ISSUE TITLE
Column name ambiguity in documentation for `classification_cost()`

### ISSUE THREAD

**Issue by jdonland** [tidymodels/yardstick#343]
The documentation for classification_cost mentions both "cost" and "costs" as the correct column name for classification costs in the costs data frame.

**Comment by EmilHvitfeldt** [comment]
Good catch, I will make sure this is clarified more before the next release.
